content/spin/v2/cli-reference.md

```diff
@@ -4169,6 +4857,34 @@ Options:
 
 {{ blockEnd }}
 
+**Note:** For additional information, please see the [Managing Templates](/spin/managing-templates) and/or [Creating Templates](/spin/template-authoring) sections of the documentation.
+
+{{ startTab "v2.7"}}
+
+<!-- @selectiveCpy -->
+
+```console
+$ spin test --help
```

Comment:
Hmm, I see this is existing, but it's a plugin and we can't make any guarantees about whether it will be present or which version.  Don't worry about it for this PR but something to come back to.